qiskit_experiments/library/quantum_volume/qv_experiment.py

```diff
@@ -103,7 +98,9 @@ def __init__(
         # Set configurable options
         self.set_experiment_options(trials=trials, seed=seed)
 
-        if not simulation_backend and HAS_SIMULATION_BACKEND:
+        if not simulation_backend and HAS_AER:
+            from qiskit import Aer
```

Comment:
I think it has been long enough that we are safe to update to this form. The qiskit import has been giving a pending deprecation warning for a year.

```suggestion
            from qiskit_aer import Aer
```